### ISSUE TITLE
Potential minor bug (maybe)

### ISSUE THREAD

**Issue by chestfindman** [ExOptsTeam/NFSUExOpts#13]
It's quite possible I mixed something up and the issue is on my end.
Or maybe it has already been discussed (but I have not been able to find it).

Certain upgrades in "Underground mode" (Career) lose their price tag (become free, as if already owned).
Specifically bumpers, side skirts, roof scoops, spoilers, hoods.
Performance upgrades, paint types and everything else does still cost money however.

Issue disappears upon removal of Extra Options, also all .ini settings remain by default (except for hotkeys, I switch all of those to "unassigned" key 232).

Why? Because I tend to switch game focus and in general multitask a lot while the game is running so I don't want to inadvertently enable something and there is no option to turn off hotkeys entirely.

And since I'm already opening an issue might as well mention the bug with "widebody kits" appearing among customization options right away (before they are unlocked), which upon mistakenly selecting them crashes the game.
This issue however I do remember reading about somewhere so it's likely already known and it's not a priority.

Apologies if I'm wasting your time with this and/or it is already known and thanks for reading.

**Comment by splashbxcv** [comment]
Yes, the "free" bug also occurs on my end, but I didn't knew it was a bug before this, because I never played through Underground without ExtOpts, so I never actually knew it was a big and thought it was just intended, also I mistakenly clicked Widebody kit several times, while fast scrolling, that's also a thing!
